### ISSUE TITLE
Lack of keyboard functionality - 2 issues

### ISSUE THREAD

**Issue by GoogleCodeExporter** [adapt-it/adaptit#52]
```
What steps will reproduce the problem?
1.  Glossing or Free Translation Mode in AdaptIt 5.1.0
2.  Attempting to enter the number 2 from my keyboard simply does not work
in the glossing or in the free translation mode.  I can get 1 and 3 and
other numbers, but the number 2 does not register.
3.  When in free translation mode, the tab key does not advance to other
buttons (as it did in previous versions of AdaptIt).  For instance, I
cannot tab to the "Next" button, but whenever I press tab, it stays on
"Shorten".

What is the expected output? What do you see instead?
I expect to be able to use the number 2.  It does not work.
I expect to be able to tab to other buttons, as opposed to using the mouse
to select the buttons each time.  The tab key moves the cursor from the
free translation window to the "Shorten" button only.


What version of the product are you using? On what operating system?
I am using AdaptIt 5.1.0 on Vista.

Please provide any additional information below.
I can copy and paste the number 2 from previously loaded adaptations or
from other word-processing programs.  It is not that the number 2 is not
able to be used, just not functioning directly as a result of a single
keystroke.
```

Original issue reported on code.google.com by `kawill...@gmail.com` on 20 Aug 2009 at 12:47

**Comment by eb1** [comment]
The behavior seems to be better in 6.5.8 / Windows 7, but the second issue reported is not completely fixed:

- Numbers 0-9 can be typed into the edit box correctly.
- Pressing the Tab key does move focus to most of the buttons in the tab order. HOWEVER, once focus goes to the Remove button it gets stuck there and pressing Tab or Shift+Tab does not move the focus to another control.

**Comment by adaptitbruce** [comment]
Hi Erik,
I've fixed this - issue two. The Remove button was the ONLY button in
the bar which had a style flag set, and it was  wxTAB_TRAVERSAL .
Ironic, that made the Remove button the only button which didn't
participate properly in tab traversal. So I removed its style so it was
unstyled as were all the others. Recompiled and tested, and now tabbing
steps through all the buttons etc. So this issue is done with.

--Bruce
